src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -498,6 +498,8 @@ public static Language ParseLanguage(string title)
             if (lowerTitle.Contains("portuguese"))
                 return Language.Portuguese;
 
+            if (lowerTitle.Contains("hungarian"))
+                return Language.Hungarian;
             var match = LanguageRegex.Match(title);
```

Comment:
Added the new line, and changed the regex.

Just for some additional info:
The new regex would not parse anything starting or ending with an _ correctly, as it is a word-character, and therefore not seeing it as the boundary.
As a result, _HUN would fail, as well as HUN_ No idea if that schema is utilized much though.

Also, as I'm Dutch, "hun" is a pretty common word in our language, so I wanted to limit [space]HUN[space] hits.

That's why I ended up enclosing HUN with the _ . or - character.